### PR TITLE
DOCS-1088: Pt3: Add to and fix cloud API

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

**Review by andf-viam — APPROVED** [review]
LGTM! Some tiny opt language suggestions, and a question about the `errors_only` flag. Looks great!!

File: docs/program/apis/cloud.md [hunk 1]
```diff
@@ -487,6 +483,250 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `robot_part_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the robot part to get logs from.
+- `filter` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Only include logs with messages that contain the string `filter`.
+  Defaults to empty string `""`, meaning no filter.
+- `dest` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Optional filepath to write the log entries to.
+- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to include error logs.
+  Defaults to `True`, including logs by default.
```
[the remark this answers is not in the document]
Comment (JessamyT) [reply docs/program/apis/cloud.md:561]: Oops good point. Not 100% sure but the way I wrote it seems wrong.
```suggestion
- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to include error logs.
  Defaults to `True`, including only error-level messages by default.
```

File: docs/program/apis/cloud.md [hunk 2]
```diff
@@ -487,6 +483,250 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+
+- `robot_part_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the robot part to retrieve logs from.
+- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to include error logs.
+  Defaults to `True`, including logs by default.
+- `filter` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Only include logs with messages that contain the string `filter`.
+  Defaults to empty string `""`, meaning no filter.
+
+**Returns:**
+
+- (_LogsStream[[List[LogEntry]]](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.LogEntry)): The asynchronous iterator receiving live robot part logs.
```
[the remark this answers is not in the document]
Comment (JessamyT) [reply docs/program/apis/cloud.md:602]: It is like this in the source code so leaving as-is unless it's idiomatically odd to include the underscore (even though it is indeed present)

File: docs/program/apis/cloud.md [hunk 3]
```diff
@@ -487,6 +483,250 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `robot_part_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the robot part to get logs from.
+- `filter` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Only include logs with messages that contain the string `filter`.
+  Defaults to empty string `""`, meaning no filter.
+- `dest` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Optional filepath to write the log entries to.
+- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to include error logs.
```
Comment (JessamyT) [inline docs/program/apis/cloud.md:560]:
```suggestion
- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to include only error logs.
```

File: docs/program/apis/cloud.md [hunk 4]
```diff
@@ -487,6 +483,250 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+
+Get an asynchronous iterator that receives live robot part logs.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `robot_part_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the robot part to retrieve logs from.
+- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to include error logs.
```
Comment (JessamyT) [inline docs/program/apis/cloud.md:595]:
```suggestion
- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to include only error logs.
```

File: docs/program/apis/cloud.md [hunk 5]
```diff
@@ -487,6 +483,250 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+  Defaults to `True`, including logs by default.
```
Comment (JessamyT) [inline docs/program/apis/cloud.md:596]:
```suggestion
  Defaults to `True`, including only error-level messages by default.
```

File: docs/program/apis/cloud.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (andf-viam) [reply docs/program/apis/cloud.md:602]: OK! Then let's leave as-is per source, thanks for checking!

File: docs/program/apis/cloud.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (andf-viam) [reply docs/program/apis/cloud.md]: I think we might also want to update `whether to include error logs` to something closer to `whether to limit returned log messages to error logs only`

File: docs/program/apis/cloud.md [hunk 6]
```diff
@@ -487,6 +483,250 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+
+Get an asynchronous iterator that receives live robot part logs.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `robot_part_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the robot part to retrieve logs from.
+- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to include only error logs.
```
Comment (JessamyT) [inline docs/program/apis/cloud.md:595]:
```suggestion
- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to limit returned log messages to error logs only.
```

File: docs/program/apis/cloud.md [hunk 7]
```diff
@@ -487,6 +483,250 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `robot_part_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the robot part to get logs from.
+- `filter` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Only include logs with messages that contain the string `filter`.
+  Defaults to empty string `""`, meaning no filter.
+- `dest` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Optional filepath to write the log entries to.
+- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to include only error logs.
```
Comment (JessamyT) [inline docs/program/apis/cloud.md:560]:
```suggestion
- `errors_only` [(bool)](https://docs.python.org/3/library/functions.html#bool): Specifies whether to limit returned log messages to error logs only.
```

File: docs/program/apis/cloud.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JessamyT) [reply docs/program/apis/cloud.md]: Done!

**Pull request merged**